src/E57XmlParser.cpp

```diff
@@ -83,6 +85,17 @@ inline int64_t convertStrToLL( const std::string &inStr )
 #endif
 }
 
+/// Parse a double according the the classic ("C") locale.
+/// \return The parsed double or 0.0 on error.
+inline double convertStrToDouble( const std::string &inStr )
```

Comment:
Could you please move this to StringFunctions and rename to `strToDouble` to parallel `floatingPointToStr`? You can keep it inline if you want.

Also the docs are generally using `@return` instead of `\return`.